src/main/java/org/primefaces/expression/impl/RowExpressionResolver.java

```diff
@@ -69,27 +70,27 @@ public String resolveClientIds(FacesContext context, UIComponent source, UICompo
                     for (UIComponent comp : column.getChildren()) {
 
                         if (clientIds.length() > 0) {
-                            clientIds += " ";
+                            clientIds.append(" ");
                         }
 
-                        clientIds += data.getClientId(context) + seperatorChar + row + seperatorChar
-                                + dynamicColumn.getId() + seperatorChar + i + seperatorChar + comp.getId();
+                        clientIds.append(data.getClientId(context) + seperatorChar + row + seperatorChar
```

Comment:
@dvmarcilio From a compiler perspective it should be okay to mix up `append` and `+` operators, since the latter will most probably get replaced automatically. However, in my opinion I wouldn't mix them anyway.

Reply:
What is the best way in your opinion? one `append()` for each `String`, in the same line? multiple lines?

Reply:
Yes, I'd use one `append` per `String`. Checkstyle will tell you its preference. :stuck_out_tongue_winking_eye: